apps/vr-tests/package.json

```diff
@@ -12,7 +12,6 @@
     "just": "just-scripts",
     "lint": "just-scripts lint",
     "screener:build": "cross-env NODE_OPTIONS=--max-old-space-size=3072 just-scripts storybook:build",
```

Comment:
`screener:build` will be renamed in a separate PR along with other miscellaneous changes